src/main/java/com/wojtek/parkingmeter/helpers/Validator.java

```diff
@@ -0,0 +1,57 @@
+package com.wojtek.parkingmeter.helpers;
+
+import com.wojtek.parkingmeter.helpers.enums.TicketType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+public class Validator {
+
+
+    public Validator() {
+    }
+
+    public static boolean validateNewTicket(String ticket_type, String nr_plate) {
+
+
+        if (TicketType.DISABLED.toString().equals(ticket_type.toUpperCase()))
+            if (nr_plate.length() == 5)
+                return true;
+
+        if (TicketType.REGULAR.toString().equals(ticket_type.toUpperCase()))
+            if (nr_plate.length() == 5)
+                return true;
+
+        return false;
+    }
+
+    public static boolean checkIfAlreadyStarted(JdbcTemplate jdbcTemplate, Long id) {
```

Comment:
jak dostaniesz `id` równe null to co się stanie?

Reply:
Zgadza się, wysypie się.